modules/postgres/examples_test.go

```diff
@@ -28,8 +34,7 @@ func ExampleRunContainer() {
 		postgres.WithUsername(dbUser),
 		postgres.WithPassword(dbPassword),
 		testcontainers.WithWaitStrategy(
-			wait.ForLog("database system is ready to accept connections").
-				WithOccurrence(2).
+			wait.ForSQL(nat.Port("5432/tcp"), "postgres", dbURL).
```

Comment:
Could you add a valid SQL to wait for? Otherwise, the default one (`SELECT 1`) will be used, and it could not be clear for casual readers. If we add here a valid, postgres-specific SQL , then it will be much clearer.

Thanks!

Reply:
@mdelapenya done